### PR TITLE
7-post-comment-to-article

### PR DESCRIPTION
**Pull request by hc-tbhc**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -167,4 +167,60 @@ describe('/api/articles/:id/comments', () => {
...
+            return request(app)
+            .post('/api/articles/3/comments')
+            .send()
+            .expect(400)
+            .then(({ body }) => {                
+                expect(body.message).toBe('400: Missing required fields');
+            })
+        })
+
+        test('400: POST /api/articles/:id/comments - attempting to post an incorrectly formatted comment will return an appropriate error', () => {
```
Comment (roisinoreilly) [inline __tests__/app.test.js:210]: great test

**Review by roisinoreilly — COMMENTED** [review]
great testing so far, you're just missing 400 and 404 tests for trying to post a comment to an invalid and non existent article id. Other than that your edge case tests are great and implementation is spot on so far, any questions let me know!
